tests/nested/manual/uc20-create-partitions-encrypt/task.yaml

```diff
@@ -87,7 +87,7 @@ execute: |
     ls /run/mnt/ubuntu-boot/EFI/ubuntu/grub.cfg
 
     umount /run/mnt/ubuntu-boot
-    umount /run/mnt/ubuntu-data
+    umount /run/mnt/data
```

Comment:
given this is create-partions, I would expect that to be /ubuntu-data still?

Reply:
ah yes you're right, to be fair I didn't actually try running this test as it's very difficult to get running apparently